package.json

```diff
@@ -129,10 +145,12 @@
     "js-base64": "^2.3.2",
     "jsonfile": "^5.0.0",
     "lodash": "^4.17.11",
+    "node-emoji": "^1.10.0",
     "open": "^6.4.0"
   },
   "devDependencies": {
     "chai": "^4.2.0",
+    "chalk": "^2.4.2",
```

Comment:
Hi @gkushang.

I think should be a dependency and not devDependency

Reply:
You’re right! Let me fix it soon in next patch version. Thanks for pointing it out !

Reply:
@lneves12 published the fix with `5.0.2`

Reply:
thanks @gkushang